web/src/main/java/org/akaza/openclinica/control/admin/CreateXformCRFVersionServlet.java

```diff
@@ -340,7 +343,7 @@ private XformContainer parseInstance(String xform, Errors errors, Html html, Str
 
             for (XformItem xformItem : xformItems) {
                 for (XformGroup xformGroup : xformGroups) {
-                    if (xformItem.getItemGroup().equals(xformGroup.getGroupName())) {
+                    if (xformItem.getItemGroup() != null && xformItem.getItemGroup().equals(xformGroup.getGroupName())) {
```

Comment:
Is it possible to make these loops more efficient? Something like this?
             for (XformItem xformItem : xformItems) {
                 if (xformItem.getItemGroup()) continue;  // this will reduce quite a few iterations in the inner loop
                 if (xformItem.getItemGroup().equals(xformGroup.getGroupName())) {